lib/block_view.go

```diff
@@ -7408,134 +7799,10 @@ func (bav *UtxoView) _connectAcceptNFTBid(
 	totalDiff := big.NewInt(0).Add(sellerPlusBidderDiff, creatorPlusCoinDiff)
 	if totalDiff.Cmp(big.NewInt(0)) > 0 {
 		return 0, 0, nil, fmt.Errorf(
-			"_connectAcceptNFTBid: Sum of participant diffs is >0 (%d, %d, %d, %d)",
+			"_helpConnectNFTSold: Sum of participant diffs is >0 (%d, %d, %d, %d)",
 			sellerDiff, bidderDiff, creatorDiff, coinDiff)
 	}
 
-	return totalInput, totalOutput, utxoOpsForTxn, nil
-}
-
-func (bav *UtxoView) _connectNFTBid(
```

Comment:
Nit: I think the code would have been easier to review if you hadn't moved the _connectNFTBid() function, which I think messed up the diff.

Reply:
It wasn't moved - just an annoying way in which GitHub displays the diff. I had to move out all the logic that "sells" the NFT so we didn't have duplicated code to handle royalties and NFT ownership.